### PR TITLE
Add iNaturalist class & import method

### PR DESCRIPTION
**Pull request by zwolf**
Adds a simple iNaturalist object that has a single method. That method sends a POST request to Panoptes that starts an iNat Observation import. It uses the base `Panoptes.client()` POST method, which includes authentication. 

~I've included two tests which are currently failing--I can't figure out why the mock isn't working. The SubjectSet test was the model and it sure seems to me that they that method should be stubbed and not actually trying to call out. It's a simple enough test that it might be easier to just remove it, but let me know if I'm missing something obvious.~

Tests are sorted, I forgot to remove some dev/debug stuff. Thanks @yuenmichelle1

### CODE REVIEW

**Review by yuenmichelle1 — CHANGES_REQUESTED** [review]
This looks fine to me. Just a couple small changes

- I think we should also include Inaturalist import in __init__.py
- Not a blocker, but should we include this within user_guide or panoptes_client.rst?

**Comment by yuenmichelle1** [comment]
Yup this looks good. 👍  I think it belongs. So that we can run `from panoptes_client import Inaturalist` in a direct manner.

File: panoptes_client/__init__.py [hunk 1]
```diff
@@ -12,3 +12,4 @@
 from panoptes_client.workflow import Workflow
 from panoptes_client.subject_workflow_status import SubjectWorkflowStatus
 from panoptes_client.caesar import Caesar
+from panoptes_client.inaturalist import Inaturalist
```
[the remark this answers is not in the document]
Comment (zwolf) [reply panoptes_client/__init__.py:15]: Required for convenient importing

**Review by yuenmichelle1 — APPROVED** [review]
lgtm

**Pull request merged**